sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -246,6 +247,7 @@ internal class BitmapWidgetRenderer(
       GLUtils.texImage2D(GLES20.GL_TEXTURE_2D, 0, it, 0)
       GLES20.glBindTexture(GLES20.GL_TEXTURE_2D, 0)
 
+      listener?.onLoaded(it)
```

Comment:
```suggestion
      bitmap.recycle()
```

https://github.com/mapbox/mapbox-maps-android/pull/1708/files#r984878027
^ Related to this comment.

Alternative is to not expose a listener and just recycle it. If the developer wants to share the bitmap across multiple `BitmapWidgetRenderer` instances, then it seems Mapbox should build a `MultiBitmapWidgetRenderer`

Reply:
Considering the desire to share a bitmap elsewhere, for example a bitmap in a View icon and the same icon for the map. But those are typically separate icons, so it does not seem like something to optimize for.

IMO if you turn this into a simple bug fix you could back port it to older versions. For example, what if you wanted to add this to the `10.4` release?

Reply:
If it is still desired to expose the `onLoaded`, can you please consider an alternative approach that will not create an incompatibility?

https://github.com/mapbox/mapbox-maps-android/pull/1708/files#r984894170

Reply:
We don't own the Bitmap, and there is nothing in the API to say that we do.  The caller could be using the Bitmap for other purposes, so recycling it in the SDK is maybe not the best approach.

Reply:
Please review latest commit

Reply:
Resolving here ! thanks for addressing